public/app/models/ListModeSwitcher.js

```diff
@@ -0,0 +1,25 @@
+(function () {
```

Comment:
This module was just pulled out from `SearchPage` for code structure consistency (with `ListSorter` and `ListPager`). Not much to see here.